README.md

```diff
@@ -89,23 +89,24 @@ The shim strives to hew closely to the C# DurableOrchestrationContext API, while
 ### API Implementation Checklist
 **Implemented**
 * `CurrentUtcDateTime`
+* `InstanceId`
+* `IsReplaying`
 * `CallActivityAsync(String name, Object input)`
 * `CreateTimer(Date fireAt)`
 * `GetInput()`
 * `WaitForExternalEvent(String name)`
 * `OrchestrationClient` binding to `string`
 
 **Not Yet Implemented**
-* `InstanceId`
-* `IsReplaying`
 * `CallActivityWithRetryAsync(String, RetryOptions, Object)`
 * `CallSubOrchestratorAsync(String, Object)`
```

Comment:
It might be too late for this feedback, but why are we suffixing these API calls with `Async`? Is there any precedent for that in JavaScript? I only know it as a .NET convention for APIs that return a Task.